src/feedback-rating/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface FeedbackRatingProp {
+    imgSrc?: string;
+    description?: string;
+    buttonLabel?: string;
+    ratings: number;
+    onRatingsChange: (value: number) => void;
+    onSubmit: () => void;
+}
+
+export interface StarsContainerProp {
```

Comment:
```suggestion
export interface FeedbackRatingStarsContainerProp {
```

Reply:
updated all file names to "FeedbackRatingStarsContainer"